src/hooks/use-previous.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useRef } from 'react';
+
+/**
+ * Get previous value (prevProps) in functional component
+ */
+const usePrevious = <T>(value: T): T => {
```

Comment:
put thi in https://github.com/Lomray-Software/client-helpers/tree/prod/src/hooks